docs/using_the_android_sdk_bom.md

```diff
@@ -0,0 +1,38 @@
+# Using the Android SDK BOM in your own projects
+
+A BOM (Bill of Materials) is used to lock the dependencies (direct and transitive) down to known compatible versions. Gradle itself does not offer such a mechanism but it is a well known and used concept in the Maven world (see [Dependency Management](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management)).
+In order to bring this functionality to Gradle the Android SDK relies on the [dependency management plugin](https://github.com/spring-gradle-plugins/dependency-management-plugin) provided by Spring. This plugin allows us to use a standard Maven BOM in a Gradle project.
+
+If your project depends on the AeroGear Android SDK you can make use of the same BOM to ensure that your dependencies are compatible with the SDK. All you need to do is to apply the plugin and reference the BOM.
+
+At the top of your gradle build file (but below `buildscripts`) add the plugin itself:
```

Comment:
I think that it will be worth to mention mobile services here and finish this procedure by importing core and sync service.

Reply:
Might be nice to say it's the `build.gradle` file, just to be explicit about the name

Reply:
I didn't want to give this example because they do not yet exist and artifact names could change

Reply:
But you're right that it would be good to show how to import a dependency without specifying a version